content/en/network_performance_monitoring/installation.md

```diff
@@ -62,29 +62,22 @@ To enable network performance monitoring with the Datadog Agent, use the followi
         enabled: true
     ```
 
-4. Start the system-probe:
+4. If you are running an Agent older than v6.18/7.18, manually start the system-probe and enable it to start on boot (since v6.18/v7.18, system-probe will start automatically when the Agent is started):
```

Comment:
```suggestion
4. If you are running an Agent older than v6.18 or 7.18, manually start the system-probe and enable it to start on boot (since v6.18/v7.18, system-probe will start automatically when the Agent is started):
```